systemtest/src/test/java/io/strimzi/systemtest/connect/ConnectST.java

```diff
@@ -1324,9 +1328,68 @@ void testConfigureDeploymentStrategy(ExtensionContext extensionContext) {
         assertThat(kafkaConnect.getSpec().getTemplate().getDeployment().getDeploymentStrategy(), is(DeploymentStrategy.ROLLING_UPDATE));
     }
 
+    @ParallelNamespaceTest
+    void testKafkaAndKafkaConnectWithJMX(ExtensionContext extensionContext) {
```

Comment:
You seem to cover both Kafka and Connect JMX in a single test which is fine AFAIC. But I wonder if there should be a separate class for JMX STs. This way it is a bit lost.

Reply:
I was thinking about it too, but I thought that will be not necessary if we'll have there single test. Let me move it to separate ST